src/module.rs

```diff
@@ -146,6 +192,25 @@ impl Module {
         FunctionValue::new(value).expect("add_function should always succeed in adding a new function")
     }
 
+    /// Gets the `Context` from which this `Module` originates.
+    /// 
+    /// # Example
+    /// ```
+    /// use inkwell::context::{Context, ContextRef};
+    /// use inkwell::module::Module;
+    /// 
+    /// let global_context = Context::get_global();
+    /// let global_module = Module::create("my_global_module");
+    /// 
+    /// assert_eq!(global_module.get_context(), global_context);
+    /// 
+    /// let local_context = Context::create();
+    /// let local_module = local_context.create_module("my_module");
+    /// let local_context = ContextRef::new(local_context);
```

Comment:
This made me realize, I don't think I meant `ContextRef::new()` to be public, as it's only really useful when returned by Inkwell. Mind fixing this? It should probably be `pub(crate)`. If you want to compare contexts here, you should be able to dereference the `local_module`'s `ContextRef` to a `Context` and compare that.

Reply:
Alright, doing it now. I thought it was strange as well.